primitives/src/cdd_id.rs

```diff
@@ -49,8 +50,8 @@ impl CddId {
     /// and `investor_uid`.
     pub fn new(did: IdentityId, investor_uid: InvestorUid) -> Self {
         let cdd_claim_data = CDDClaimData {
-            investor_did: RawData(did.to_bytes()),
-            investor_unique_id: RawData(investor_uid.0),
+            investor_did: Scalar::from_bits(did.to_bytes()),
+            investor_unique_id: Scalar::from_bits(investor_uid.0),
         };
```

Comment:
These changes are unrelated to the topic of this PR. The cryptography library and Polymesh were out of sync and I needed to do this to make the code compile.

Reply:
It might have been resolved after https://github.com/PolymathNetwork/Polymesh/pull/499 went in, did you rebase on the latest `develop`?